helpers/controllers/submissionController.ts

```diff
@@ -43,12 +45,53 @@ export const createSubmission = async (
   }
 }
 
-export const submissions = (_parent: void, arg: Submission, _ctx: Context) => {
+export const submissions = async (
+  _parent: void,
+  arg: ArgsGetSubmissions,
+  _ctx: Context
+) => {
   const { lessonId } = arg
-  return Submission.findAll({
+  const subs = await Submission.findAll({
     where: {
       status: 'open',
       lessonId
     }
   })
+  if (subs.length === 0) {
+    return subs
+  }
+  const userIds = subs.reduce((acc: Set<number>, sub: any) => {
```

Comment:
none of these should be necessary, you just use an `include` field in your submission query.